co-log-core/src/Colog/Core/Action.hs

```diff
@@ -575,3 +579,10 @@ multiplicate (LogAction l) = LogAction $ \msgs -> l (fold msgs)
 {-# INLINE multiplicate #-}
 {-# SPECIALIZE multiplicate :: Monoid msg => LogAction m msg -> LogAction m [msg] #-}
 {-# SPECIALIZE multiplicate :: Monoid msg => LogAction m msg -> LogAction m (NonEmpty msg) #-}
+
+{- | Allows changing the internal monadic action -}
+hoistLogAction
```

Comment:
The example of this function is actually very simple. Let's say that you have pure logger action and you want to log all messages inside `IO`. So you have the following function:

```haskell
perfromPureLogsInIO :: PureLogger a -> IO a
```

You can then convert action that does logging in a pure monad to action that performs logging in the `IO` monad.

Reply:
Thanks for clarification, I'll add this into the docs! :)